src/libs/ReportUtils.js

```diff
@@ -1,4 +1,4 @@
-import _ from 'underscore';
+import _, {isString} from 'underscore';
```

Comment:
Previously in this PR I think you used `_.isString` so I think we can do the same here, instead of specifically grabbing that function from the import